src/tools/_tools.line-height.scss

```diff
@@ -21,7 +21,7 @@
 ///   our `$FLUIDMS-GLOBAL-BASELINE`.
 @function fluidms-line-height($scale, $ratio) {
     // Calculate the default line-height (in `rem`).
-    $_default-line-height: (fluidms-pow($ratio, $scale)) * $FLUIDMS-GLOBAL-LINE-HEIGHT;
+    $_default-line-height: (fluidms-pow($ratio, $scale)) * ($FLUIDMS-GLOBAL-LINE-HEIGHT / fluidms-pow($FLUIDMS-GLOBAL-LINE-HEIGHT-RATIO, $scale));
```

Comment:
Not part of this PR but concerning the variables I'd recomment to remove the `GLOBAL` from the variable name as the uppercase symbolizes the global scope (document that)

Reply:
1. I agree that this is not part of this PR.
2. I’m not sure if I agree with removing the `GLOBAL` part from the variable(s). Yes, we have a convention that says, uppercase vars are global, but IMO it is more clear to leave the global namespace in there, to make that fact more clear. If you want, we can discuss this separately in a dedicated issue.